code/modules/mob/living/carbon/alien/humanoid/caste/praetorian.dm

```diff
@@ -38,10 +38,17 @@
 	if(node.recent_queen_death)
 		user << "<span class='danger'>You are still too burdened with guilt to evolve into a queen.</span>"
 		return 0
-	if(!alien_type_present(/mob/living/carbon/alien/humanoid/royal/queen))
+	if(!alien_type_present(/mob/living/carbon/alien/humanoid/royal/queen) && (!(user.ckey in list("optimumtact", "coiax", "technoalchemist", "crimsonvision", "misterperson", "mrstonedone", "jordie0608", "xhuis", "iamgoofball", "cheridan", "pkpenguin321", "basilman"))))
 		var/mob/living/carbon/alien/humanoid/royal/queen/new_xeno = new (user.loc)
 		user.alien_evolve(new_xeno)
 		return 1
 	else
```

Comment:
The way this if statement is setup, one of the people on this list could become a queen even if there is already one.